pandas/core/groupby/generic.py

```diff
@@ -1357,7 +1357,13 @@ def arr_func(bvalues: ArrayLike) -> ArrayLike:
 
         # We could use `mgr.apply` here and not have to set_axis, but
         #  we would have to do shape gymnastics for ArrayManager compat
-        res_mgr = mgr.grouped_reduce(arr_func, ignore_failures=True)
+        try:
+            res_mgr = mgr.grouped_reduce(
+                arr_func, ignore_failures=numeric_only is lib.no_default
+            )
+        except NotImplementedError as err:
+            msg = f"{how} is not supported for at least one provided dtype"
```

Comment:
why not just let the original exception surface?  wont that include dtype information?

Reply:
Thanks! That's much better.